docs/src/zh/guide/advanced/module-versioning.md

```
@@ -103,106 +109,184 @@ MAJOR，哪怕这次发布只是一次源码没动的重新构建——而下面
-- **是** → 它应该在移除清单上。如果不在，请开 issue；那是策略层面的疏漏，不只是你的麻烦。
-- **否** → 试上面那条重建路径（抬 pin → 重编 → 抬 `api-version`）。如果**重编就编不过**，
-  说明这次变更连源码兼容一起破了，那你面对的是一次迁移，不是一次重建。
+| 构建时 pin | 跑在框架 6.2.0 | 跑在框架 6.2.1+ |
+|---|---|---|
+| 6.2.0 | 正常 | **`NoSuchMethodError`** —— 找 `(AbstractDataEntity)`，已不存在 |
+| 6.2.1 | **`NoSuchMethodError`** —— 找 `(BaseDataEntity)`，尚不存在 | 正常 |
 
-[jls13]: https://docs.oracle.com/javase/specs/jls/se21/html/jls-13.html
+所以「针对新框架重编一下」不是一个能让老服务器留在原地的修复。它只是换了哪一边能跑。
+两个格子失败的原因相同、方向相反，这正是为什么修法取决于缺失符号写的是**哪一代**类型
+——见[怎么读输出](#怎么读输出-两类成因-修法相反)。
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)